include/library/spdm_requester_lib.h

```diff
@@ -254,6 +254,7 @@ libspdm_return_t libspdm_get_certificate_choose_length_ex(void *spdm_context,
  * this function also perform the basic mutual authentication.
  *
  * @param  spdm_context           A pointer to the SPDM context.
+ * @param  reserved               Reserved for session_id.
```

Comment:
I would say "Reserved for session_id and is ignored." Ie, they can pass a pointer or `NULL` but it won't have any effect.

Reply:
That is more appropriate indeed, the doxygen comment is updated.